release_notes/unreleased.md

```diff
@@ -1 +1,2 @@
 **Unreleased**
+* Removed 'pyc' files from the app tarball
```

Comment:
Please mention ticket ID in the release notes.

Reply:
Done